dashboard.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+require 'functions.php';
+
+$db = displayPDO();
+
+    $stmt = $db->query("SELECT `title` ,`intro_text` FROM `intro`");
+    $title_array = $stmt->fetch();
+    $stmt = $db->query("SELECT `paragraph_1`, `image`, `paragraph_2` FROM `about_me`");
+    $about_me_array = $stmt->fetch();
+    $stmt = $db->query("SELECT `title`, `image`, `paragraph` FROM `projects`");
+    $projects_array = $stmt->fetch();
+
+    $data = array_merge($title_array, $about_me_array, $projects_array);
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Dashboard</title>
+</head>
+<body>
+    <h1>DASHBOARD PAGE</h1>
+        <a class="nav-text" href="index.php">Home Page</a>
+            <form action="upload_intro.php" method="POST" enctype="multipart/form-data">
+                <h3>Intro:</h3>
+                    Title text:         <input name="title_text" type="text" value="<?php echo $title_array["title"] ?>"><br><br>
```

Comment:
wrap title text and intro text in label tags

Reply:
Like this? 

<label>Title text:</label>
<label>Intro text:</label>

Reply:
Like this?

//<label>Title text:</label>
//<label>Intro text:</label>

Reply:
Like this?

`//<label>Title text:</label>
//<label>Intro text:</label>`

Reply:
Like this?

`<label>Title text:</label>
<label>Intro text:</label>`